style.css

```diff
@@ -0,0 +1,69 @@
+body{
+    margin: 0;
+    background-repeat:no-repeat;
+    background-size: cover;
+    background-image: linear-gradient(rgba(58, 58, 158, 0.8), rgb(186, 113, 187)), url("https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg");
+    height: 230vh;
+    opacity: 0.8;
+    padding: 0px;
+    color: whitesmoke;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+header{
+    color: #fff;
+    text-align: center;
+}
+#container{
+    background-color:#4a4444;
+    /* margin: 0 200px; */
```

Comment:
I kindly request that you remove all the commented code as it would improve the cleanliness and readability of your code.